examples/simple-console/src/main/java/com/snowplowanalytics/Main.java

```diff
@@ -64,7 +64,7 @@ public static void main(String[] args) throws InterruptedException {
                 Collections.singletonMap("foo", "bar")));
 
         // This is an example of a eventSubject for adding user data
-        Subject eventSubject = Subject.builder().build();
+        Subject eventSubject = new Subject(new SubjectConfiguration());
```

Comment:
Just realizing that we don't have an empty constructor for Subject, could be useful for this case when you don't want to use the configuration to set up Subject properties.